src/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System
+{
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Enum | AttributeTargets.Delegate, Inherited = false)]
+    public sealed class SerializableAttribute : Attribute 
+    {
+        public SerializableAttribute() {
+        }
```

Comment:
Wait, something strange is going on. My branch has the fix: https://github.com/KrzysztofCwalina/corefx/blob/BinarySerialization/src/System.Runtime.Serialization.Formatters/src/System/SerializableAttribute.cs
But it's not being picked up by upstream/master